Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs

```diff
@@ -32,25 +39,28 @@ public void ConfigureServices(IServiceCollection services)
 
             var connection = @"Server=dev.retrotest.co.za;Database=ereader;User Id=group4;Password=3bHNuE8&rvG+99U2;";
             services.AddDbContext<MyDbContext>
-                (options => options.UseSqlServer(connection));
+                (options => options.UseSqlServer(connection)); //other option: UseSqlServer(Configuration.GetSection(connection))
```

Comment:
So this commented out code (GC) should then be something likes 
*// TODO: other option: UseSqlServer(Configuration.GetSection(connection))*
If you really wanna keep it, same applies for line 50

Reply:
thank you I've incorporated your advice.

Reply:
You're welcome, once its changed I'll approve :)